client/src/BookProfile/BookProfile.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from "react";
+import data from '../Mocks/books.json';
+import { Card } from 'antd';
+import './BookProfile.css';
+import Rater from 'react-rater';
+import { withRouter } from "react-router-dom";
+import defaultBook from '../pictures/bookSmile.png';
+
+const { Meta } = Card;
+const defaultBookId = 1;
+
+export class BookProfile extends Component {
+
+    constructor(props) {
+        super(props);
+
+        let splitedPath = window.location.pathname.split("/");
+        let bookId = defaultBookId;
+        if(splitedPath.length === 3) bookId = parseInt(splitedPath[2]);
+
+        this.state = {
+            bookId: bookId,
+            bookData: undefined,
+        }
+    }
+
+    componentDidMount() {
+        this.loadBookData();
+    }
+
+    loadBookData() {
+        let bookData = Object.keys(data).filter(item => { return data[item].id === this.state.bookId })[0];
+        bookData = data[bookData];
+
+        if(bookData.image.length === 0) bookData.image = defaultBook;
```

Comment:
I would recommend using curly braces everywhere. Anyway, is that some kind of standard to not using it?

Reply:
Well if it's needed with one line after condition I can add it. Is there standard to use it ?

Reply:
Uncle Bob - Clean Code

Reply:
Good to know

Reply:
https://www.c-sharpcorner.com/UploadFile/d0e913/lame-question-of-the-day-role-of-curly-braces-in-our-cod/